upload-and-update/departments.json

```diff
@@ -1,7 +1,41 @@
-{
-	"ME": "Mechanical Engineering",
-	"RT": "Rubber Technology Center",
-	"PH": "Physics",
-	"BT": "Biotechnology",
-	"MA": "Mathematics"
-}
+{  
+   "AE": "Aerospace Engineering",
+   "AG": "Agricultural & Food Engineering",
+   "AR": "Architecture & Regional Planning",
+   "BT": "Biotechnology",
+   "CH": "Chemical Engineering",
+   "CY": "Chemistry",
+   "CE": "Civil Engineering",
+   "CS": "Computer Science & Engineering",
+   "CR": "Cryogenic Engineering",
+   "ED": "Center for Educational Technology",
+   "EE": "Electrical Engineering",
+   "EC": "Electronics & Electrical Communication Engineering",
+   "GS": "G S Sanyal School of Telecommunications",
+   "GG": "Geology & Geophysics",
+   "HS": "Humanities & Social Sciences",
+   "IM": "Industrial & Systems Engineering",
+   "IT": "Information Technology",
+   "MS": "Materials Science",
+   "MA": "Mathematics",
+   "ME": "Mechanical Engineering",
+   "MD": "Medical Science & Technology",
+   "MT": "Metallurgical & Materials Engineering",
+   "MI": "Mining Engineering",
+   "NA": "Ocean Engineering & Naval Architecture",
+   "N2": "Oceans, Rivers, Atmosphere and Land Sciences",
```

Comment:
@icyflame : Yeah, I guess you can change that to ```CL```, there were other departments also, which had different ID's in the hyperlinks, which I changed manually, I didn't know about this, but this seems right.